packages/serverless/src/model/database/user-group.ts

```diff
@@ -21,11 +23,23 @@ export class ModelUserGroup extends ModelGeneral<DocumentUserGroup> {
     super(databaseName, ModelUserGroup.collectionName);
   }
 
+  public async getGroupInfo(groupName: string): Promise<TGroupInfo> {
+    const modelGroup = new ModelGroup(this.databaseName);
+    const group = await modelGroup.findOne({ groupName });
+    if (!group) {
+      throw new Error('Group not existed');
+    }
+    return {
+      ...group,
+      members: await (await this.find({ groupId: group._id })).toArray(),
```

Comment:
I would prefer conjunct group and it's user in the domain layer.

Reply:
Moved to domain layer